### DISCUSSION TITLE
require even more uniq package names

### DISCUSSION THREAD

**Discussion by jimmywarting** [npm/feedback#491] [Registry] [answered]
I know you already do some smart uniqueness lookup (maybe like removing `-.`?)

But recently i installed a suspicious package i tought was the right one ending with `____js`
turned out there where two packages with the same name. only differences being that the other package ended with "js"

can you filter that one out when you compare if they are similar eg: `util`, `utiljs` and `util.js`?

**Comment by MylesBorins** [comment, marked as answer]
@jimmywarting this is a great idea, such a great idea that our team is already working on it! I hope you understand that I can't share too many details as we don't want people gaming the system.
